rm/rm-node-updater/src/main/java/org/ow2/proactive/resourcemanager/updater/RMNodeUpdater.java

```diff
@@ -194,6 +194,7 @@ private long getRemoteLastModifiedMillis(String url) {
      * Check if the node jar is up-to-date and download it
      * @return true if the local jar is up-to-date or has been successfully renewed. false in case of error
      */
+    @SuppressWarnings({ "squid:squid:S2095" })
     private boolean makeNodeUpToDate() {
         switch (isLocalJarUpToDate(nodeJarUrl, nodeJarSaveAs)) {
             case OUT_DATED:
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') End this switch case with an unconditional break, return or throw statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS128)